x-pack/plugins/lists/server/scripts/exception_lists/new/exception_list_item_with_list.json

```diff
@@ -0,0 +1,23 @@
+{
+  "item_id": "endpoint_list_item_lg_val_list",
```

Comment:
Would it make sense to specify a `list_id` here to make it work out of the box with the scripts? It looks like the other `exception_list_item_*.json` snippets specify the `list_id` as well. Without it the default behavior of the following will return a `400` -- at which point you have to go in and add it and wait for the server to restart because of the modification.

``` 
$ ./post_exception_list_item.sh ./exception_lists/new/exception_list_item_with_list.json
{
  "statusCode": 400,
  "error": "Bad Request",
  "message": "[request body]: Invalid value \"undefined\" supplied to \"list_id\""
}
```

Reply:
Updated! Thank you!